src/EPR.Calculator.API/Builder/Summary/CalcResultSummaryBuilder.cs

```diff
@@ -17,12 +17,18 @@ public class CalcResultSummaryBuilder : ICalcResultSummaryBuilder
         private const int MaterialsBreakdownHeaderInitialColumnIndex = 5;
         private const int MaterialsBreakdownHeaderIncrementalColumnIndex = 11;
         private const int DisposalFeeSummaryColumnIndex = 93;
-        private const int LaDataPrepCostsSection4ColumnIndex = 216;
+        private const int LaDataPrepCostsSection4ColumnIndex = 202;
         private const int MaterialsBreakdownHeaderCommsInitialColumnIndex = 100;
         private const int MaterialsBreakdownHeaderCommsIncrementalColumnIndex = 9;
         //Section-(1) & (2a)
         private const int decimalRoundUp = 2;
         private const int DisposalFeeCommsCostsHeaderInitialColumnIndex = 179;
+        //Section-(1) & (2a)
+        private const int Total1Plus2ABadDebt = 193;
+        //Section-3
+        private const int SAOperatingCostCostsHeaderInitialColumnIndex = 195;
```

Comment:
Is it correct Index? You need to leave indexes for other sections..

Reply:
i guess that will keep changing Mazar but in my downloaded current csv it is showing currently
![image](https://github.com/user-attachments/assets/e962feb1-abde-4f98-910e-215735c4e480)

Reply:
when you are doing your section better to leave indexes for others so that you should not change it later. You can count it from result file and fix your column index. If you wont change now again you need change once others code merged and it will be rework for you..